package.json

```diff
@@ -172,6 +170,8 @@
     "webpack.vote": "https://github.com/webpack/voting-app.git"
   },
   "resolutions": {
-    "remark-responsive-tables": "git://github.com/chenxsan/remark-responsive-tables.git#bugfix/fix-empty-head"
+    "remark-responsive-tables": "git://github.com/chenxsan/remark-responsive-tables.git#bugfix/fix-empty-head",
+    "sitemap-static/minimist": "1.2.5",
```

Comment:
Why are the last 2 necessary?

Reply:
These two lock their version of `minimist`, and there're vulnerabilities https://snyk.io/vuln/npm:minimist in lower version hence I upgrade them to the latest version.